ztp/kube-compare-reference/required/machine-config/ssh/75-limit-users-ssh-access-master.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 75-master-limit-users-ssh-access
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=Limit Users' SSH Access
+            Before=sshd.service
+            [Service]
+            Type=oneshot
+            RemainAfterExit=yes
+            ExecStart=/bin/bash -c 'allowed=$(grep -rPi \'^\h*(allow|deny)(users|groups)\h+\H+(\h+.*)?$\' /etc/ssh/sshd_config*); \
+                        if [ "$allowed" == "" ]; then \
+                        echo "AllowUsers  core"  >> /etc/ssh/sshd_config; fi'
```

Comment:
Editing sshd_config seems dangerous.  Can you just drop a new file in sshd_config.d?

Reply:
Good catch, fixed. Thanks!